views/partials/protect_yourself.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="aligner">
+<h2>Protect yourself from hackers.<br />Start here.</h2>
+{{#if breach }}
+  <p class="demi">See if your account was exposed in the {{ breach.name }} breach or other data breaches.</p>
+{{else}}
+  <p class="demi">The first step to keeping your online accounts safe is knowing what you’re up against. Enter your email to find out if your accounts have been compromised.</p>
+{{/if}}
+<form action="/scan" method="post">
+    <div class="input-group">
+        <input class="input-group-field email-to-hash" type="email" name="email" placeholder="Enter Email Address">
+        <div class="input-group-button">
+            <img src="img/Search.svg" alt="search icon" />
+            <input id="submit-email" type="submit" class="button" value="Search">
+        </div>
+    </div>
+    <span>Your email will not be stored. Find out more in our <a href="
+    https://www.mozilla.org/privacy/firefox/" alt="Mozilla Firefox privacy policy">Privacy Policy.</a></span>
```

Comment:
I _think_ we can lose the `alt=""` here, since I'm not sure that is supported on an `<a>` tag.
Or not sure if we meant `title=""` or `aria-label=""` (or whatever).